ansible/module_utils/port_utils.py

```diff
@@ -349,7 +349,7 @@ def get_port_alias_to_name_map(hwsku, asic_name=None):
         elif hwsku == "Cisco-8102-C64":
             for i in range(0, 64):
                 port_alias_to_name_map["etp%d" % i] = "Ethernet%d" % (i * 4)
-        elif hwsku in ["Cisco-8101-O32", "Cisco-8111-C32", "Cisco-8111-O32"]:
+        elif hwsku in ["Cisco-8101-O32", "Cisco-8101-32FH-O-C01", "Cisco-8111-C32", "Cisco-8111-O32"]:
```

Comment:
this one didn't follow the hwsku naming convention, we need to ask vendor to address it

Reply:
ACK